running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,214 @@
+# Getting started
+Rent a Ubuntu Server 18.04 or higher from a trusted hosting company or install for the sake of learning experience Ubuntu Server 18.04 or higher in VirtualBox.
+
+Once you're set we will start by accessing our server. If you're on windows you should get putty(https://www.putty.org/), if you're on macOS or Linux you can use the terminal which is standard available. With putty just follow the screen ( insert your domain/ip-address and click ok ) for macOS and linux users type:
+
+```bash
+ssh username@domain/ipaddress
+```
+`username example => root
+domain/ipaddress example => yourdomain.com or 127.0.0.1`
+
+Now you will be prompted for your password (if in putty you will be prompted for the password as soon you click on Open) Insert your password and presh enter (you will not see what you type, so make sure you type your password in the right order). Now we are in our server :)
+
+## Update and Upgrade the system
+The first thing we will do is updating our Ubuntu Server with the following command
+```bash
+sudo apt update && sudo apt upgrade -y
+```
+We are using 'sudo' since I do not know if you're running as root user or as regular user. When you're logged in as regular user. No problem! Just enter your password when the terminal asks for your password. With the command above you will update the connected repository's the server is connected to and when the server is done updating the repository's it will directly upgrade all the packages which were updated.
+
+# Installing the webserver
+
+Now we will get to the more difficult part of the entire server setup, installing nginx, php-fpm, mariadb-server with phpmyadmin.
+```bash
+sudo apt install nginx php-fpm mariadb-server -y
+```
+The installation process is now triggered.When this is finished we will install phpmyadmin to handle our database server
+```bash
+sudo apt install phpmyadmin -y
+```
+Now we are set to get some configurations going!
+
+## Configuring the webserver
+
+First we want to know which version of php-fpm is installed we will check this by writing the following command in the terminal
+```bash
+sudo php -v
+```
+The output will be looking similar to this:
+```
+PHP 7.2.24-0ubuntu0.18.04.3 (cli) (built: Feb 11 2020 15:55:52) ( NTS )
+Copyright (c) 1997-2018 The PHP Group
+Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
+    with Zend OPcache v7.2.24-0ubuntu0.18.04.3, Copyright (c) 1999-2018, by Zend Technologies
+```
+
+Second we want to do is enabling php in our nginx server and setting the configuration just as we want. So we will be going to edit our nginx config. Now we need to install a command line text editor called "vim". After that we installed it we will directly open the configuration file.
+```bash
+sudo apt install vim -y && sudo vim /etc/nginx/sites-available/default
```

Comment:
No need to chain commands, just put them in two separate lines.
Explaining what `&&` does is unnecessary here, so let's just not use it.